### ISSUE TITLE
Auto-pickup is ineffective for the loot window when using the Pick Pocket and Pick Lock spells.

### ISSUE THREAD

**Issue by jilinge2** [balakethelock/SuperAPI#3]
I swapped the positions of 0 and 1 here. When picking up items from a corpse, the auto-pickup feature works, but the auto-pickup does not work when using the Rogue's Pick Pocket and Pick Lock spells.
`function IfShiftAutoloot()
	if IsShiftKeyDown() then 
		SetAutoloot(0)
	else
		SetAutoloot(1)
	end
end`

**Comment by balakethelock** [comment]
are you using vanillatweaks autoloot together with superwow

**Comment by jilinge2** [comment]
No, I haven't used it simultaneously because some servers will prompt "Unable to verify the game version, this may be due to the file being corrupted" when using VanillaTweaks.

**Comment by bordarino** [comment]
Anyone get this to work? I try LootSlot(slotid, 1) function, but i dont know how to made it work
